src/styles/widget.scss

```diff
@@ -21,52 +21,44 @@
 
   &__container {
     display: block;
-    margin-left: 10px;
+    margin-left: 8px;
 
     @media (min-width: 430px) {
       display: flex;
-      align-items: center;
+      align-items: flex-start;
       width: 100%;
       justify-content: space-between;
-      margin-right: 10px;
+      margin-right: 8px;
     }
 
     .button {
-      margin-top: 5px;
-
-      @media (min-width: 430px) {
-        margin-top: 0;
-      }
+      margin-top: 4px;
     }
   }
 
   &__text {
     display: flex;
     flex-direction: column;
-    padding-right: 5px;
+    padding-right: 8px;
   }
 
   &__title {
     font-size: 14px;
-    font-weight: 200;
+    font-weight: 300;
     letter-spacing: -0.03em;
+    line-height: 14px;
     margin: 0;
   }
 
   &__title + &__subtitle {
     font-size: 12px;
-    font-weight: 200;
+    font-weight: 300;
     letter-spacing: -0.03em;
-    line-height: 13px;
-    margin-top: 2px;
+    line-height: 14px;
   }
 
   &__iconbird {
-    margin: 0;
-
-    @media (min-width: 430px) {
-      margin: 9px 0 0 0;
-    }
+    margin: 5px 0 0 0;
```

Comment:
The logo gets a fixed height from top
![image](https://user-images.githubusercontent.com/63985172/135029779-a04d68bf-002f-40b6-800f-7620a992a3d4.png)